Dockerfile

```diff
@@ -37,15 +44,19 @@ ENV HADOOP_CONF_DIR ${HADOOP_HOME}/etc/hadoop
 # Setup Hive
 ENV HIVE_CONF_DIR ${HADOOP_CONF_DIR}
 
-# Setup spark
+# Setup Spark
+ENV SPARK_16_HOME=/opt/spark-${SPARK_16_VERSION}
+ENV SPARK_20_HOME=/opt/spark-${SPARK_20_VERSION}
 ENV SPARK_HOME=/opt/spark-${SPARK_VERSION}
 ENV PYTHONPATH=${SPARK_HOME}/python:${SPARK_HOME}/python/lib/py4j-0.10.3-src.zip
 ENV PYSPARK_PYTHON=python
 ENV PATH=$PATH:${SPARK_HOME}/bin
 
-# Add these two spark packages when submitting PySpark applications
-#ENV PYSPARK_SUBMIT_ARGS="--packages com.databricks:spark-csv_2.11:1.4.0,com.databricks:spark-avro_2.10:2.0.1,graphframes:graphframes:0.1.0-spark1.6 pyspark-shell"
-ENV PYSPARK_SUBMIT_ARGS="--packages com.databricks:spark-csv_2.11:1.5.0,com.databricks:spark-avro_2.11:3.1.0,graphframes:graphframes:0.3.0-spark2.0-s_2.11 pyspark-shell"
+# Add these two spark packages when submitting PySpark applications (Spark 1.6)
+ENV PYSPARK_SUBMIT_ARGS="--packages com.databricks:spark-csv_2.10:1.5.0,com.databricks:spark-avro_2.10:2.0.1,graphframes:graphframes:0.1.0-spark1.6 pyspark-shell"
```

Comment:
Is `PYSPARK_SUBMIT_ARGS` different for 1.6 and 2.0? 

Not sure how to deal with different environment for different spark versions... Maybe the same thing you did with `SPARK_VERSION` above.